cachito/web/api_v1.py

```diff
@@ -5,10 +5,12 @@
 import tempfile
 from collections import OrderedDict
 from copy import deepcopy
-from typing import Any, Dict
+from datetime import date, timedelta
+from typing import Any, Dict, Optional
 
 import flask
 import kombu.exceptions
+import pydantic
```

Comment:
Have we settled on using pydantic for validation? LGTM, I don't have strong opinions on this, but I heard folks commeting about using JsonSchema, so I'll leave this thread open in case anyone wants to chime in. :)

Reply:
pydantic is not agreed, but the PR can be a good starting point for such discussion, since there is visible a real case.